scripts/discourse-sync/main.py

```diff
@@ -1,26 +1,31 @@
 import os
 import sys
 import yaml
-from pydiscourse import DiscourseClient
+from client import Client
```

Comment:
It's probably better if this is a relative import, in case you have some other thing called `client` in the path.

```suggestion
from .client import Client
```

Reply:
Didn't work for me:
```
ImportError: attempted relative import with no known parent package
```

Reply:
You probably need an empty `scripts/discourse-sync/__init__.py` file so that Python knows that `discourse-sync` is a package.

Reply:
Ah yeah, I didn't think about how this is running. It's not important, the chances of a conflict in this specific case are super low.